test/components/avatar_component_test.rb

```diff
@@ -2,6 +2,8 @@
 
 class AvatarComponentTest < ViewComponent::TestCase
   include ActionView::Helpers::AssetUrlHelper
+  include ActionView::Helpers::UrlHelper
+  include Rails.application.routes.url_helpers
```

Comment:
needed those lines ignorer to use `url_for` not sure if someone has a better approach for that